Packs/UpdateEntriesBySearch/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### SetTagsBySearch
+- Added the `filter_user_type` argument to improve filtering.
+- Had entries be modified only for updated them.
```

Comment:
not sure i understand what you wanted to say here, can you clarify please?

Reply:
In older versions, all the entries matched were updated with `setEntriesTags` even in unchanged entries.
For example, there are 2 entries, A and B. A has tag xyz. B doesn't have tag xyz.
When trying to add the `xyz` tag to entries and A and B are matched with the pattern, both A and B are updated with `setEntriesTags` even though A doesn't need to be updated in older version. But in the new version, only B is updated.

![image](https://user-images.githubusercontent.com/54964121/127110193-241fffc8-4e8d-412e-aa89-968b6af96d50.png)

Reply:
ok so can you please improve the second bullet in the release notes a bit so it will be clear?

Reply:
Sure, I've updated the release note.